config/kernel-monitor.json

```diff
@@ -10,6 +10,11 @@
 			"type": "KernelDeadlock",
 			"reason": "KernelHasNoDeadlock",
 			"message": "kernel has no deadlock"
+		},
+		{
+			"type": "HardwareErrorFatal",
```

Comment:
`Hardware` is a very general term. Given this is only for UEFI, can we name it more specifically?

Reply:
@wangzhen127 How about CperHardwareErrorFatal?
CPER stands for Common Platform Error Record and is defined as an industry standard in the [UEFI Specification](https://uefi.org/specifications).